parglare/parser.py

```diff
@@ -461,9 +463,23 @@ def _skipws(self, context):
                     level=1)
 
     def _next_token(self, context):
+        tokens = self._next_tokens(context)
+        if len(tokens) == 0:
```

Comment:
It is more "Pythonic" when checking emptyness to use boolean evaluation, just `if not tokens:`.